libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -38,7 +38,7 @@
 public class PolicyManagerTests extends ESTestCase {
     /**
      * A module you can use for test cases that don't actually care about the
-     * entitlements module.
+     * entitlement module.
```

Comment:
Oh I forgot we went with the singular here. I suppose I made this same mistake in several places...